.travis.yml

```diff
@@ -51,10 +54,18 @@ cache:
     - $HOME/nanomsg
 
 install:
-  - export P4_NAME="test"
-  - export P4_PREFIX="test"
-  - export P4_PATH="`pwd`/bmv2/tests/test.p4"
-  - ./autogen.sh; ./configure 'CXX=g++-4.8' --with-bmv2; make -j2
+  - ./autogen.sh; 
 
 script:
+  - export P4_NAME="pdtest"
+  - export P4_PREFIX="pdtest"
+  - export P4_PATH="`pwd`/bmv2/tests/pdtest/pdtest.p4"
+  - ./configure 'CXX=g++-4.8' --with-bmv2; make -j2
   - make check
+  - make clean
+  - export P4_NAME="oftest"
+  - export P4_PREFIX="oftest"
+  - export P4_PATH="`pwd`/bmv2/tests/openflow/oftest.p4"
+  - export P4C_BM_FLAGS="--plugin of --openflow-mapping-dir `pwd`/bmv2/tests/openflow --openflow-mapping-mod oftest"
+  - ./configure 'CXX=g++-4.8' --with-bmv2 --with-of; make -j2 CFLAGS="-I$HOME/p4ofagent/include"
+  - make check CFLAGS="-I$HOME/p4ofagent/include -L$HOME/p4ofagent/lib"
```

Comment:
can you just use the `LDFLAGS` and `CPPFLAGS` exported above (add your directories to the variables)?